config/routes.rb

```diff
@@ -22,5 +22,13 @@
   get '*path', to: 'home#app'
   get '/*undefined', to: redirect('/')
 
+  namespace :admin do
+    resources :teams do
+      collection do
+        get :export_csv
+      end
+    end
+  end
```

Comment:
Please check that this is necessary. When running `$ rails routes` this is generating on lines 145-152:
```
GET        /admin/teams/export_csv(.:format)                                                                 admin/teams#export_csv
GET        /admin/teams(.:format)                                                                            admin/teams#index
POST       /admin/teams(.:format)                                                                            admin/teams#create
GET        /admin/teams/new(.:format)                                                                        admin/teams#new
GET        /admin/teams/:id/edit(.:format)                                                                   admin/teams#edit
GET        /admin/teams/:id(.:format)                                                                        admin/teams#show
PATCH      /admin/teams/:id(.:format)                                                                        admin/teams#update
PUT        /admin/teams/:id(.:format)                                                                        admin/teams#update
DELETE     /admin/teams/:id(.:format)               
```

while on line 72 this gets generated as part of `ActiveAdmin.routes(self)`:
```
export_csv_admin_teams GET        /admin/teams/export_csv(.:format)                                                                 admin/teams#export_csv
```